res/openvpn-gui-res-en.rc

```diff
@@ -435,9 +436,11 @@ BEGIN
     IDS_ERR_ACCESS_SERVICE_PIPE "Access to service pipe failed."
     IDS_ERR_WRITE_SERVICE_PIPE "Writing to service pipe failed."
     IDS_ERR_INSTALL_ISERVICE     """OpenVPNServiceInteractive"" is not installed.\n"
-                                 "Tasks requiring administrative access may not work."
+                                 "Wintun driver and tasks requiring administrative access may not work."
     IDS_ERR_NOTSTARTED_ISERVICE  """OpenVPNServiceInteractive"" is not started.\n"
-                                 "Tasks requiring administrative access may not work."
+                                 "Wintun driver and tasks requiring administrative access may not work."
+    IDS_ERR_NOTSTARTED_ISERVICE_ADM  """OpenVPNServiceInteractive"" is not started.\n"
+                                     "Wintun driver will not work."
```

Comment:
Please copy this added resource IDS_ERR_NOTSTARTED_ISERVICE_ADM to all language files so that future translators will take note.
Its hard search and add missing strings later.

Reply:
will do.